dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/QueryOperator.java

```diff
@@ -51,6 +51,8 @@ public enum QueryOperator {
   IN("in", true),
   SW("sw"),
   EW("ew"),
+  NULL("??"),
```

Comment:
```sql
eventdatavalues->'D7m8vpzxHDJ' is null
```

as well as `not null` works. I would favor using the `null` sql operator as it can be reused for non jsonb as well.

Reply:
yes, this is better, thanks